src/settings/BatchGroupConfigurationSettings/ExportVoucherButton/ExportVoucherButton.js

```diff
@@ -59,6 +61,7 @@ const ExportVoucherButton = ({ batchVoucherId, format, stripes, fileName }) => {
         data-test-batch-voucher-export-download
         icon="download"
         onClick={downloadBatchVouchers}
+        aria-labelledby={intl.formatMessage({ id: 'ui-invoice.settings.batchGroupConfiguration.download' })}
```

Comment:
Did you mean `aria-label`? As I see this button will use a label of an element with `id` equals `"Download"` or `"Herunterladen"` (e.g. for German locale), I'm not sure it's what is expected.

Reply:
And AFAIK, `IconButton` uses `icon` prop as `aria-label` by default